src/main/resources/assets/js/components/DeploymentComponent.jsx

```diff
@@ -23,8 +23,10 @@ define([
       this.setState({loading: bool});
     },
 
-    handleDestroyDeployment: function () {
-      this.props.destroyDeployment(this.props.model, this);
+    handleDestroyDeployment: function(forceStop) {
+      return function() {
+        this.props.destroyDeployment(this.props.model, {forceStop: forceStop}, this);
+      }.bind(this);
```

Comment:
Arh! I didn't see that you were calling the function in the onClick expression.

In stead of that solution you can bind it to pass arguments:
```
onClick={this.handleDestroyDeployment.bind(this, true)}
```
and
```
onClick={this.handleDestroyDeployment}
```
And then keep this:
```
handleDestroyDeployment: function(forceStop) {
  this.props.destroyDeployment(this.props.model, {forceStop: forceStop}, this);
},
```